src/lua/fiber.c

```diff
@@ -527,6 +542,17 @@ lbox_fiber_new(struct lua_State *L)
 	return 1;
 }
 
+struct fiber *
+get_fiber(struct lua_State *L)
```

Comment:
This must be `static`. Also there is clearly a naming convention in this file, and 2 similar getters: `lbox_checkfiber()` and `lbox_fiber_id()`. I suggest you to conform, and name it similarly. For instance, `lbox_get_fiber()`.

Reply:
Fixed